src/test/resources/features/15-attachment/attachments.feature

```diff
@@ -0,0 +1,19 @@
+ Feature: Attachments Management
+
+   Background:
+     Given a user logs in, searches a patient and visits Attachment page 
+     And a user clicks on Attachment link
+     Then the system loads Attachment page 
+     
+   Scenario: Uploading Attachment
+     And a user attaches a file
+     And a user enters a caption
+     Then the system enables upload button
+     When a user clicks the upload button 
+     Then the system uploads the attached file
+     
+   Scenario: Removing un intended Attachement 
+     And a user attaches a file or enters a caption
+     And a user clicks clear forms button
+     Then the system clears the form
+     And close browser 
```

Comment:
you have forgotten to get rid of the last step, we are taking care of it in the framework and through @After("selenium)

Reply:
Oops! I shall be more smart next time. Have refactored the codebase